invokeai/app/invocations/model.py

```diff
@@ -262,6 +262,103 @@ def invoke(self, context: InvocationContext) -> LoraLoaderOutput:
         return output
 
 
+class SDXLLoraLoaderOutput(BaseInvocationOutput):
+    """Model loader output"""
+
+    # fmt: off
+    type: Literal["sdxl_lora_loader_output"] = "sdxl_lora_loader_output"
+
+    unet: Optional[UNetField] = Field(default=None, description="UNet submodel")
+    clip: Optional[ClipField] = Field(default=None, description="Tokenizer and text_encoder submodels")
+    clip2: Optional[ClipField] = Field(default=None, description="Tokenizer2 and text_encoder2 submodels")
```

Comment:
Why are the output models `Optional`?

Reply:
Because inputs optional too, user can input only some of this models.